canopy_main.ml

```diff
@@ -61,9 +61,9 @@ module Main  (C: CONSOLE) (S: STACKV4) (RES: Resolver_lwt.S) (CON: Conduit_mirag
     store_ops.Canopy_dispatch.update () >>= fun l ->
```

Comment:
here we trash the cache, pull, and fill the cache again... also, we `update_atom ()`... but actually we only need to `update_atom ()` (since we've pulled + filled 15 lines above)